Lib/test/test_ast/test_ast.py

```diff
@@ -2303,7 +2303,7 @@ def get_load_const(self, tree):
         co = compile(tree, '<string>', 'exec')
         consts = []
         for instr in dis.get_instructions(co):
-            if instr.opname == 'LOAD_CONST' or instr.opname == 'RETURN_CONST':
+            if instr.opname in ('LOAD_CONST', 'LOAD_CONST_IMMORTAL', 'LOAD_INT'):
```

Comment:
I think this should work:

```suggestion
            if dis.opmap[instr.opname] in dis.hasconst:
```